File: Client/skladiste.js
import { SProizvod } from "./sproizvod.js"

export class Skladiste
{
    constructor(velicina, id, broj)
    {
        this.sproizvodi = [];
        this.velicina = velicina;
        this.id = id;
        this.broj = broj;
        this.container = null; 
        this.sprContainer = null;    
    }

    
}

File: Models/Skladiste.cs
using System.Collections.Generic;
using System.ComponentModel.DataAnnotations;
using System.ComponentModel.DataAnnotations.Schema;
using System.Text.Json.Serialization;

namespace Models
{
    [Table("Skladiste")]
    public class Skladiste
    {
        [Key]
        [Column("ID")]
        public int ID { get; set; }

        [Column("Broj")]
        public int Broj { get; set; }

        [Column("Velicina")]
        public int Velicina { get; set; }

       
        public virtual List<SProizvod> SProizvod { get; set; }

        [JsonIgnore]
        public Supermarket Supermarket { get; set; }
    }
}